openlibrary/templates/books/add.html

```diff
@@ -57,6 +57,15 @@ <h1>$_("Add a book to Open Library")</h1>
             </div>
         </div>
 
+        <div class="formElement">
```

Comment:
We do not want this field when inserting a new book, so you can remove it from here and it's backing server code.

We want it to appear in the book editing page. Likely somewhere near the bottom of openlibrary/templates/books/edit/about.html .